streampark-console/streampark-console-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
Could this file be renamed `casdoor.properties`, it seems to be a casdoor-only configuration

Reply:
Is there additional configuration to enable it, enforcing it is not expected, and we want the user to have the choice.